internal/agent/hooks/hook.go

```diff
@@ -17,9 +17,7 @@ var AfterInstall = []Interface{
 	&Lifecycle{}, // Handles poweroff/reboot by config options
 }
 
-var AfterReset = []Interface{
```

Comment:
so I assume we don't re-encrypt anymore? Isn't elemental-cli deleting and recreating them?

Reply:
I have no idea how this works but cli is finding it, and formatting it, but only the mapped device (so /dev/mapper/sda5 for example) which means that the encryption is still in place no? Its only formatting the decrypted device, while the encrypted one hasnt changed.

Again, no idea, but its like erasing the contents of it only.

Reply:
Yup that sounds good then 👍